student.rb

```diff
@@ -1,14 +1,18 @@
 require_relative 'person'
 
 class Student < Person
-  attr_accessor :classroom
-
   def initialize(classroom, name = 'Unknown', age = nil, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
-    @classroom = classroom
+    classroom&.add_student(self)
   end
```

Comment:
- Instead of deleting the attr_accessor :classroom, make your code more readable by removing 
` classroom&.add_student(self)` and  `@classroom&.students&.delete(self)` and update to:


```ruby
class Student < Person
  attr_accessor :classroom

  def initialize(classroom, name = 'Unknown', age = nil, parent_permission: true)
    super(name, age. parent_permission: parent_permission)
    @classroom = classroom
  end

  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) or // classroom.students.push(self) unless classroom.students.include?(self)

  end

  def play_hooky
    "¯\(ツ)/¯"
  end
end
```

Reply:
@zieeco The reason I changed it was because it ran me into a linter error. When I change it back I get the same error.
![Screenshot 2023-08-03 034333](https://github.com/ClintonYounge/School-Libraby-Project/assets/118696200/75332cb3-c30d-4116-88d6-a221a07ab983)

Reply:
I guess I could just change the name of the method.

Reply:
@zieeco What do you recommend doing?